src/models/Language.test.js

```diff
@@ -62,4 +62,13 @@ describe(`Language`, () => {
 
   });
 
+  describe(`Language.prototype.iso`, () => {
+    const lang = new Language();
+    it(`Language.prototype.iso`, function() {
+      expect(() => { lang.iso = `ctm`; }).not.toThrow();
+      expect(() => { lang.iso = `en`; }).toThrow();//e => e.name === `ISOCodeError`);
+    });
+
+  });
```

Comment:
Since there's only one test for the `iso` property we can just use an `it` statement rather than `describe`. `const lang = new Language;` will be inside the `it` statement. (PS: Notice how you don't need the `()` after a `new` operator if you're not passing data to it.)